app/models/edition.rb

```diff
@@ -49,10 +49,22 @@ class Edition < ActiveRecord::Base
   }
 
   scope :with_title_containing, -> *keywords {
```

Comment:
Looking through the codebase, I can't actually see anywhere where this scope is used with more than one keyword(s). In theory, the signature could lose the splat operator and the method simplified without affecting its usage. Worth doing?